net-misc/asterisk-g729/asterisk-g729-16.0.3.1.9.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit multilib
```

Comment:
Hmm, does this use multilib at all?

Reply:
$(get_libdir) previously required inherit multilib.  Removed.